doc/source/ray-core/handling-dependencies.rst

```diff
@@ -402,6 +402,40 @@ Example:
   {"pip": ["torch", "ray[serve]"],
   "env_vars": {"A": "a", "B": "new", "C", "c"}}
 
+.. _runtime-env-faq:
+
+Frequently Asked Questions
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Are environments installed on every node?
+"""""""""""""""""""""""""""""""""""""""""
+
+If specifying your runtime environment for the job, in ``ray.init(runtime_env=...)``, then the environment will be installed on every node.  See :ref:`Per-Job <rte-per-job>` for more details.
+
+When is the environment installed?
+""""""""""""""""""""""""""""""""""
+
+When specified per-job, the environment is installed when you call ``ray.init()``.  
+When specified per-task or per-actor, the environment is called when the task is invoked or the actor is instantiated (i.e. when you call ``my_task.remote()`` or ``my_actor.remote()``.)
+See :ref:`Per-Job <rte-per-job>` :ref:`Per-Task/Actor, within a job <rte-per-task-actor>` for more details.
+
+Where are the environments cached?
+""""""""""""""""""""""""""""""""""
+
+Any local files downloaded by the envs are cached at ``/tmp/ray/session_latest/runtime_resources``.
+
+
+
+How long does it take to install or to load from cache?
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The install time usually mostly consists of the time it takes to run ``pip install`` or ``conda create`` / ``conda activate``, or to upload/download a ``working_dir``, depending on which ``runtime_env`` options you're using. 
+This could take seconds or minutes.  
+
+On the other hand, loading a runtime environment from the cache should be nearly as fast as the ordinary Ray worker startup time, which is on the order of a few seconds.  A new Ray worker is started for every Ray actor, and for every Ray task that requires a new runtime environment.
```

Comment:
> A new Ray worker is started for every Ray actor, and for every Ray task that requires a new runtime environment.

Does this mean a new Ray worker is started for
* Every Ray Actor (no matter what)
* Every Ray Task that requires a new runtime environment

It's a bit unclear whether the Ray Actor should require a new runtime_env.

Also, a nit– there's two spaces between the start of this sentence ("A new Ray worker...") and the the period before it. We should make it one space.

Reply:
Yes that's unclear, clarified